client/src/helpers/APICalls/formDataAPI.ts

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+const formDataAPI = async (file: any) => {
```

Comment:
Maybe rename this for increased semantics? formDataAPI sounds very generic, maybe we can call it uploadImage or something similar?